AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx

```diff
@@ -32,14 +32,12 @@ export default class TodaysCodePage extends React.Component {
         var moscow = london.clone().tz('Europe/Moscow');
```

Comment:
Are these edits to the home page relevant/ where they needed?

Reply:
Yes, these were needed to pass npm linting